source/index.html

```diff
@@ -267,26 +235,32 @@ <h3 class="contacts-item__title">Приезжайте в гости:</h3>
     </a>
 
     <ul class="site-contacts">
-      <li class="site-contact">
+      <li class="site-contacts__contact">
```

Comment:
Обычно применяют более абстрактное:
```suggestion
      <li class="site-contacts__item">
```